### PR TITLE
Fix svg2pdf doc

### PR DESCRIPTION
**Pull request by armstrongnate**
[ignore-commit-lint]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "danger": "danger",
     "export-translations": "./scripts/translations/export.js",
     "import-translations": "./scripts/translations/import.js",
-    "svg2pdf": "./scripts/svg2pdf.js",
+    "svg-to-pdf": "./scripts/svg2pdf.js",
```
Comment (armstrongnate) [inline package.json]: @vanwagonet I think this is how you envisioned it? https://github.com/instructure/canvas-ios/blob/master/scripts/svg2pdf.js#L9

Comment (vanwagonet) [reply package.json]: oh, I originally had it as svg-to-pdf and renamed it and forgot to update the comment. lazy typing wanted no dashes.

Comment (armstrongnate) [reply package.json]: Okay I'll change the comment then. 😄

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.14% | 95.16% | -0.01%
**Student** | 92.84% | 92.84% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.71% | 92.71% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**